src/core/components/hv-screen/index.js

```diff
@@ -344,15 +344,15 @@ export default class HvScreen extends React.Component {
   /**
    *
    */
-  onUpdate = (href, action, currentElement, opts) => {
+  onUpdate = (href, action, currentElement, opts, target) => {
```

Comment:
I think this function is called all over the place by custom behaviors, so I think we might want to make it a key of the `opts` object instead, so that we don't break compatibility